test/AaveTest/PositionManager.spec.ts

```diff
@@ -7,18 +7,26 @@ const hre = require('hardhat');
 const UniswapV3Factoryjson = require('@uniswap/v3-core/artifacts/contracts/UniswapV3Factory.sol/UniswapV3Factory.json');
```

Comment:
Why is the PositionManager test now in the AaveTest folder? 
can we name folders consistently?